dotCMS/src/main/resources/apps/dotAnalytics-config.yml

```diff
@@ -0,0 +1,48 @@
+name: "Analytics Integration"
+iconUrl: "https://static.dotcms.com/assets/icons/apps/saml.png"
+allowExtraParameters: true
+description: "Analytics integration provides the ability to, given a Client Id and Client Secret, resolve an access token to be injected across all the analytics services included the resolving of the Analytics Key (Id)."
+
+params:
+  clientId:
+    value: true
+    hidden: false
+    type: "STRING"
+    label: "Analytics Client ID"
+    hint: "Client ID is required to resolve access token from IDP"
+    required: true
+  clientSecret:
+    value: true
+    hidden: true
```

Comment:
should be hidden?

Reply:
The secret should be hidden as it is right now.